js/checkout.js

```diff
@@ -180,6 +200,76 @@ class Blockonomics {
     redirect_to_finish_order() {
         window.location.href = this.data.finish_order_url
     }
+
+    _set_refresh_loading(loading=false) {
+        if(loading) {
+            this._refresh.querySelector('.blockonomics-icon-refresh').classList.add('spin')
+            this._refresh.setAttribute('disabled', 'disabled')
+        } else {
+            this._refresh.querySelector('.blockonomics-icon-refresh').classList.remove('spin')
+            this._refresh.removeAttribute('disabled')
+        }
+    }
+
+    refresh_order() {
+        let url = `${this.data.api_url}?get_order=${this.get_url_param('show_order')}&crypto=${this.get_url_param('crypto')}`
+
+        this._set_refresh_loading(true)
+
+        // Stop Progress Counter
+        clearInterval(this.progress.interval)
+        
+        fetch(url, {method: 'GET'}).then(
+            res => {
+                this._set_refresh_loading(false)
+                if (res.status == 200) {
+                    res.json().then(data => {
+                        this._update_order_params(data)
+                    },
+                    () => {
+                        this._fallback_refresh_order()
```

Comment:
Let's remove the fallback functions from this PR as this adds overhead

Reply:
Let's keep the one where there is network error, it helps the plugin fallback to legacy mode in case of Issues like CORS.

Reply:
Done